apps/cancel-request/translations/src/en/validation.json

```diff
@@ -2,9 +2,12 @@
     "cnc-main-applicant-full-name": {
         "required": "Enter the main applicant's full name",
         "maxlength": "Applicant's full name must be 3 and 250 characters",
-        "": "Applicant's full name must be 3 and 250 characters"
+        "minlength": "Applicant's full name must be 3 and 250 characters"
```

Comment:
Add a notUrl validation message, 'Your answer must not contain URLs or links' is what's normally used on other TLF projects 

Similar to below; it might need to say 'Main applicants...' so it's consistent? Check with UCD :)